runtime/src/parachains.rs

```diff
@@ -1213,6 +1241,26 @@ mod tests {
 		}
 	}
 
+	fn init_block() {
+		println!("Initializing {}", System::block_number());
```

Comment:
there are some committed `println!` here and elsewhere

Reply:
they're in tests and quite handy in case things break for whatever reason.